src/piton/extractors/omop.py

```diff
@@ -0,0 +1,205 @@
+"""A class and program for converting OMOP v5 sources to piton."""
+
+from __future__ import annotations
+
+import dataclasses
+import datetime
+from typing import Mapping, Optional, Sequence
+
+from piton import Event
+from piton.extractors.csv import CSVExtractor
+
+OMOP_BIRTH = 4216316
+
+
+class _DemographicsConverter(CSVExtractor):
+    """Convert the OMOP demographics table to events."""
+
+    def get_patient_id_field(self) -> str:
+        return "person_id"
+
+    def get_file_prefix(self) -> str:
+        return "person"
+
+    def get_events(self, row: Mapping[str, str]) -> Sequence[Event]:
+        if row["birth_datetime"]:
+            birth = datetime.datetime.fromisoformat(row["birth_datetime"])
+        else:
+            year = 1900
+            month = 1
+            day = 1
+
+            if row["year_of_birth"]:
+                year = int(row["year_of_birth"])
+            else:
+                raise RuntimeError(
+                    "Should always have at least a year of birth?"
+                )
+
+            if row["month_of_birth"]:
+                month = int(row["month_of_birth"])
+
+            if row["day_of_birth"]:
+                day = int(row["day_of_birth"])
+
+            birth = datetime.datetime(year=year, month=month, day=day)
+
+        return [
+            # 4216316 is the OMOP birth code
+            Event(start=birth, code=4216316, event_type=row["load_table_id"])
```

Comment:
Maybe I am a bit confused here, but when I look up for birth on Athena Ohdsi, birth has Concept ID of 4216316 and concept code of 3950001. Here, we are noting it as code.